tpm2_pytss/crypto.py

```diff
@@ -88,6 +88,11 @@ def key_from_encoding(data):
         key = cert.public_key()
     elif sdata.startswith(b"-----BEGIN PUBLIC KEY-----"):
         key = load_pem_public_key(data, backend=default_backend())
+    elif sdata.startswith(b"-----BEGIN RSA PRIVATE KEY-----") or sdata.startswith(
+        b"-----BEGIN EC PRIVATE KEY-----"
```

Comment:
Not a fix now type of thing, but IIRC, scissor lines in PEM are not required unless you have N>1 things in a single file. So it would probably be best to just try/except the whole thing like the DER code.

Reply:
So more or less skip the sdata.startswith and just do try ... except pass?

Reply:
Yeah but you don't have to put it in this PR, ill take it without it.